### PR TITLE
🚑 [Cosmostation] Fix missing account request

### PR DESCRIPTION
**Pull request by nwingt**
Related to https://github.com/likecoin/liker-land/issues/649

### CODE REVIEW

File: src/utils/cosmostation.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const getCosmostationExtensionOfflineSigner = (
 ): OfflineSigner => ({
   getAccounts: async () => {
     const response = await window.cosmostation.cosmos.request({
-      method: 'cos_account',
+      method: 'cos_requestAccount',
```
Comment (williamchong) [inline src/utils/cosmostation.ts:15]: should we try catch the old code and only request when it throw?

**Review by williamchong — COMMENTED** [review]
how did it work before...

File: src/utils/cosmostation.ts (hunk 1 above)
Comment (nwingt) [reply src/utils/cosmostation.ts:15]: I tested, `cos_requestAccount` is basically the same UX if the account request has been approved

**Comment by nwingt** [comment]
> how did it work before...

Guess LikeCoin chain was not in the official supported chains

**Review by williamchong — APPROVED** [review]

**Pull request merged**